recipes/head_node_slurm_config.rb

```diff
@@ -134,15 +134,15 @@
 end
 
 file "/var/log/parallelcluster/slurm_resume.log" do
-  owner node['cluster']['slurm']['user']
-  group node['cluster']['slurm']['group']
+  owner node['cluster']['cluster_admin_user']
+  group node['cluster']['cluster_admin_group']
   mode '0644'
 end
 
 template "#{node['cluster']['slurm_plugin_dir']}/parallelcluster_slurm_resume.conf" do
   source 'slurm/parallelcluster_slurm_resume.conf.erb'
-  owner node['cluster']['slurm']['user']
-  group node['cluster']['slurm']['group']
+  owner 'root'
+  group 'root'
```

Comment:
why root for these? since suspend and resume (the only programs that read this config i think) are now run as the cluster admin user, shouldn't be owned by that user and group?

Reply:
I agree with you.
I've preferred to leave root in this iteration because I saw the same approach in other config files, e.g. slurm config is owned by root even if slurm is run as slurm user, so I thought it was kind of standard team approach.

Fixing it.

Reply:
I see other misalignment like this in other part of the recipes. My proposal is to fix in this PR the ones related to suspend/resume and then double-check every permissions on every dir/file in another PR.